tests/e2e/types.go

```diff
@@ -0,0 +1,52 @@
+package e2e
```

Comment:
(For some reason my file level commend does not work this morning.)

**Optional**: IMO this other than the version type which I commented, the rest of the file would fit into `k8s.go` with the `dependencyCRDsType` and the functions because we are doing k8s related ops with those.

I'm fine to leave this here if others agree it's more clear this way, but it's weird for me.